core/src/main/scala/cats/EitherUtil.scala

```diff
@@ -0,0 +1,14 @@
+package cats
+
+/**
+ * Convenience methods and values for Either.
+ */
+private[cats] object EitherUtil {
+  def leftCast[A, B, C](right: Right[A, B]): Either[C, B] =
+    right.asInstanceOf[Either[C, B]]
+  def rightCast[A, B, C](left: Left[A, B]): Either[A, C] =
+    left.asInstanceOf[Either[A, C]]
+
+  private[cats] val unit = Right(())
+  private[cats] val leftUnit = Left(())
```

Comment:
Why do we need private cats here and on the object?

Reply:
Inertia. We don't.